rosdep/python.yaml

```
@@ -7194,6 +7194,13 @@ python3-yaml:
       packages: [pyyaml]
   rhel: ['python%{python3_pkgversion}-PyYAML']
   ubuntu: [python3-yaml]
+python3-yappi:
+  debian: [python3-yappi]
+  fedora: [python3-yappi]
+  gentoo: [dev-python/yappi]
+  osx:
+    pip: [yappi]
+  ubuntu: [python3-yappi]
```

This isn't available for any Ubuntu releases older than Focal.
```suggestion
  ubuntu:
    '*': [python3-yappi]
    bionic: null
    xenial: null
```